declarations.d.ts

```diff
@@ -1,6 +1,303 @@
 declare module '*.json';
 declare module 'webpack-cli';
+declare module 'tapable' {
```

Comment:
this should go away once this is merged
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24527